sets/spike/prime/hand-controlled-grabber/README.md

```diff
@@ -0,0 +1,31 @@
+---
+permalink: /projects/sets/spike/prime/hand-controlled-grabber/
+title: "LEGO® Education SPIKE™ Prime: Hand-Controlled Grabber"
+maintainer:
+    user: "TheVinhLuong102"
+    name: "The Lương-Phạm Family"
+image:
+    local: "hand-controlled-grabber.png"
+    credit: "LEGO"
+video:
+    youtube: "https://www.youtube.com/embed/8uyx5npo4LA"
+description:
+    "Use the Grabber to pick up objects and bring them around!"
+building_instructions:
+    external: https://education.lego.com/v3/assets/blt293eea581807678a/blt56a81c75560c9a81/5f8802cbf71916144453a493/supercleaup-bi-pdf-book1of3.pdf
+---
+
+
+# LEGO® Education SPIKE™ Prime: Hand-Controlled Grabber
+
+The program requires PyBricks v3 firmware installed on the Prime Hub.
+
+Building instructions can be found at https://education.lego.com/en-us/product-resources/spike-prime/downloads/building-instructions.
+
+Press the Force Sensor to grab objects, and release the Force Sensor to let go of them.
+
+The programming code is as follows:
+
+```python
```

Comment:
We made a new include that adds a copy button to make it easy to copy the code. It can be added like this:

```suggestion
{% include copy-code.html %}
```python
```